### PR TITLE
Fix the root level of logging

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

**Comment by myronkscott** [comment]
added a small commit for a debugging tool

File: client-runtime/src/main/java/com/tc/cmdline/Probe.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (chrisdennis) [inline client-runtime/src/main/java/com/tc/cmdline/Probe.java:1]: Was this an accidental pickup?

**Comment by myronkscott** [comment]
Separate commit for a debugging tool that I want to get in.
________________________________

**Comment by myronkscott** [comment]
obsolete

**Pull request closed without merging**